app/views/points/_case_form.html.erb

```diff
@@ -53,4 +55,12 @@
   document.getElementById('trigger-id-grab').addEventListener('click', function() {
     document.getElementById('case').style.display = "none";
   });
+
+  document.getElementById('case-text').addEventListener('change', function() {
+    var to_parse = document.getElementById('case-text').childNodes[1].value;
+    var id = parseInt(to_parse)
+    var desc = document.getElementById('case-text').firstElementChild[id].attributes[0].nodeValue;
```

Comment:
I'm not familiar with the `.attributes` property, but [according to MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/attributes):

> the Attr nodes' indexes may differ among browsers

Thus, simply selecting element `[0]` might not work in every browser. I'm assuming you want to access the `data-title` attribute? If so, I think you could do so by using `.dataset.title` instead of `.attributes[0].nodeValue`.

Reply:
This is still the wrong version.

Reply:
Oh wait! My bad. That form is not supposed to exist anymore. Meant to delete it.